### DISCUSSION TITLE
emoji parse

### DISCUSSION THREAD

**Discussion by chocovelod** [markedjs/marked#2242] [Q&A]
Hey everyone! How can i parse emoji in marked?

**Comment by UziTech** [comment]
Can you give an example? marked is for parsing markdown and emoji isn't really part of markdown. marked will just see emoji as text.

**Comment by menukaonline** [comment]
I doubt whether you mean about black and white emojis. if it is the problem, I think this would be helpful for you (and also for the maintainers if something can be done from their side to get rid of this problem): https://www.reddit.com/r/firefox/comments/jnysqk/some_emojis_are_typing_black_and_white_if_using/

**Reply by UziTech** [reply]
That issue doesn't seem related to marked.
